cloud/functions.js

```diff
@@ -0,0 +1,14 @@
+Parse.Cloud.define('hello', (req) => {
+  req.log.info(req);
+  return 'Hi';
+});
+
+Parse.Cloud.define('asyncFunction', async (req) => {
+  await new Promise((resolve) => setTimeout(resolve,1000));
+  req.log.info(req);
+  return 'Hi async';
+});
+
+Parse.Cloud.beforeSave('Test', () => {
+  throw new Parse.Error(212,'Saving test objects is not available.')
```

Comment:
Not sure we should use Error 212 here, this may be a number that will be occupied but a new Parse Error in the future, see https://github.com/parse-community/parse-server/issues/6477#issuecomment-637901549. I suggest to use an error code >=9000.